reckoner/meta.py

```diff
@@ -27,4 +27,4 @@
     from pkgutil import get_data
     _raw_ver = get_data(__distribution_name__, 'version.txt').decode('UTF-8', 'ignore').rstrip("\r\n")
     __version__ = re.sub(__version_modifier__, r'\g<1>-\g<2>', _raw_ver)
-__author__ = 'ReactiveOps, Inc.'
+__author__ = 'Fairwinds, Inc.'
```

Comment:
We're copy writing to `FairwindsOps Inc` and authoring to `Fairwinds, Inc.` -- I actually don't know what is the best one to use here (if there are legal implications)

Reply:
Good catch! The comma screwed up my regex :) fixed.